### DISCUSSION TITLE
能調整ＰＣＬ顯示語言嗎？還是我能協助翻譯？

### DISCUSSION THREAD

**Discussion by andy40403** [Meloong-Git/PCL#2037] [提问] [answered]
### 還是有那裡可以調整語言？好像沒有欸！
※真心希望ＰＣＬ打開能顯示繁體中文的

**Comment by Chiloven945** [comment]
多語言被拒絕了（）
主要是因為佈局的原因，如果將啟動器切換為其他語言的話可能會出現文本超出或者其他問題，所以說沒有考慮
我也有說過可以先試試繁中化看看，因為从簡中到繁中不太容易會出現上面說的情況www
當然還有因為下載站的原因

**Reply by Chiloven945** [reply]
#119

**Reply by andy40403** [reply]
@Chiloven945  謝謝

**Reply by Chiloven945** [reply, marked as answer]
回來説一下，現在本地化已經在進行中，預計2.8.5實裝
#4145

**Reply by MoYuan-CN** [reply]
如果您可以幫上忙的話，可以咨詢 @tangge233，#4145 這個拉取請求就是在做這件事情
同時可以加入我們的 Discord 伺服器，鏈接是 https://discord.gg/Eg3TzQM7yB

**Comment by tangge233** [comment]
http://weblate.tangge233.top/engage/PCL
